lib/takeaway.rb

```diff
@@ -0,0 +1,35 @@
+class Takeaway
+
+  attr_reader :menu, :my_order
+
```

Comment:
I think the naming is really good since it is easy to understand what you are trying to implement with your code.
I would suggest to have different class for Menu in separate file since it would be easier to navigate if your Takeaway class would grow and have more functionality in the future. This keeps your project better oragnized and makes it easier for someone else to navigate your structure.
https://sourcemaking.com/refactoring/extract-class